font-icons.html

```diff
@@ -38,8 +38,8 @@
       --lumo-icons-dropdown: "\ea18";
       --lumo-icons-edit: "\ea19";
       --lumo-icons-error: "\ea1a";
-      --lumo-icons-eye: "\ea1b";
-      --lumo-icons-eye-disabled: "\ea1c";
+      --lumo-icons-eye-disabled: "\ea1b";
+      --lumo-icons-eye: "\ea1c";
```

Comment:
@jouni any ideas why a fresh build would flip these specific icons?

I wonder if it's because of platform-specific glob result order? OS X orders `--lumo-icons-eye` first, Linux doesn't.

Reply:
Probably fixed by https://github.com/vaadin/vaadin-lumo-styles/pull/88/commits/7915e3867aad444e3c829d92e923a783556c314a

Reply:
Hi! Thanks for the work here.

Not sure about the order, need to test. Looks like I was the last one to commit the font-icons.html file, so that was generated on macOS.

Reply:
At least with your last update (“glob sort”, didn’t test without that commit), the order is preserved correctly for me.

Reply:
That `gulp-sort` solution ended up being useless, as the final glyph order was still decided by `gulp-iconfont` (for whatever reason). Update pushed, I think it looks good now.

Reply:
Yeah, the order looks good for me as well. The resulting font on the other hand doesn’t 😅 Left a comment about that.

Reply:
We might be tangled up with https://github.com/nfroidure/svgicons2svgfont/blob/master/src/iconsdir.js :(

Reply:
I think finally figured it all out. Element order sanity required bumping `gulp-iconfont`.